docs/esp32/quickref.rst

```diff
@@ -442,14 +442,27 @@ Legacy methods:
 
     Equivalent to ``ADC.block().init(bits=bits)``.
 
+.. method:: ADC.deinit()
+
+    Provided to deinit the adc driver.
```

Comment:
This documentation for `ADC.deinit()` has split the documentation for `ADC.width()`, suggest moving it down around line 466